packages/dashboard/src/components/cardGallery/components.js

```diff
@@ -17,114 +17,89 @@
 import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
-import { themeHelpers } from '@web-stories-wp/design-system';
+import { Button, BUTTON_TYPES } from '@web-stories-wp/design-system';
 
+const pictureCss = css`
+  picture {
+    display: block;
+    & > img {
+      display: block;
+      height: 100%;
+      width: 100%;
+      object-fit: fill;
+      border-radius: ${({ theme }) => theme.borders.radius.medium};
+      border: ${({ theme }) =>
+        `1px solid ${theme.colors.border.defaultNormal}`};
+    }
+  }
+`;
 export const GalleryContainer = styled.div`
   flex: 1;
   display: flex;
   justify-content: space-between;
   align-items: start;
 `;
 
-export const MiniCardButton = styled.button(
-  ({ theme }) => css`
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100%;
-    width: 100%;
-    display: block;
+export const DisplayPage = styled.div(
+  ({ $isThreeRows }) => css`
     overflow: hidden;
-    cursor: pointer;
-    padding: 0;
-    background-color: transparent;
-    border: 1px solid ${theme.colors.border.defaultNormal};
-    border-radius: ${theme.borders.radius.small};
-    ${themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
+    width: ${$isThreeRows ? '45.2%' : '38.6%'};
+    margin-left: ${$isThreeRows ? '2.4%' : '2.1%'};
+
+    ${pictureCss}
   `
 );
-MiniCardButton.propTypes = {
-  isSelected: PropTypes.bool,
-};
-
-export const DisplayPage = styled.div`
-  border: 1px solid ${({ theme }) => theme.colors.border.defaultNormal};
-  border-radius: ${({ theme }) => theme.borders.radius.small};
-  overflow: hidden;
-  ${({ theme }) =>
-    theme.numRows === 3
-      ? css`
-          width: 45.2%;
-          margin-left: 2.4%;
-        `
-      : css`
-          width: 38.6%;
-          margin-left: 2.1%;
-        `}
-`;
-
-const AspectSetter = styled.div`
-  position: relative;
-  height: 0;
-  padding-bottom: ${({ aspect }) => aspect * 100}%;
-`;
-const AspectInner = styled.div`
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-`;
-
-export const AspectRatio = forwardRef(function AspectRatio(
-  { aspect = 1, children },
-  ref
-) {
-  return (
-    <AspectSetter aspect={aspect}>
-      <AspectInner ref={ref}>{children}</AspectInner>
-    </AspectSetter>
-  );
-});
-
-AspectRatio.propTypes = {
-  aspect: PropTypes.number,
-  children: PropTypes.node,
+DisplayPage.propTypes = {
+  $isThreeRows: PropTypes.bool,
 };
 
 export const Thumbnails = styled.div`
   position: relative;
-  display: flex;
-  flex-wrap: wrap;
-  align-items: start;
-  ${({ theme }) =>
-    theme.numRows === 3
-      ? css`
-          width: 52.3%;
-        `
-      : css`
-          width: 59.3%;
-        `}
+  display: grid;
+  width: 52.3%;
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 8px;
+  row-gap: 16px;
+  margin-right: 16px;
+
+  @media screen and (min-width: 1600px) {
+    width: 59.3%;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
I think this should be 4 columns at large monitor sizes instead of 2?
```suggestion
    grid-template-columns: repeat(4, 1fr);
```
<img width="1440" alt="Screen Shot 2021-08-27 at 8 08 09 AM" src="https://user-images.githubusercontent.com/35983235/131132124-b0a85d1c-7fdc-4b40-814e-a87ff4ea61b0.png">